### DISCUSSION TITLE
NET9

### DISCUSSION THREAD

**Discussion by LiorBanai** [Analogy-LogViewer/Analogy.LogViewer#2171] [Releases]
- Add NET9
 - Remove NET6

<hr /><em>This discussion was created from the release <a href='https://github.com/Analogy-LogViewer/Analogy.LogViewer/releases/tag/v7.1.0'>NET9</a>.</em>

**Comment by LevYas** [comment]
Hey! Thank you for the great tool!
Are there instruments or ways to work with correlated logs? Like by clicking on the entry press something like "show related items" and it will show all the records with current spanId and records, where the current SpanId is parent.
I mean records with these fields defined:
![image](https://github.com/user-attachments/assets/8ca37f3b-de2a-4149-b5c4-2dbbe450427b)

I use Serilog's Compact JSON format for logs with SerilogTracing.
I thought "Loggers Tree" should help, but this ribbon item is completely empty for me, only the "about" section is present.

**Reply by LiorBanai** [reply]
Interesting idea. Need to think how to visualize it. Currently there isn't sometime like this. The logger tree allows to filter namespaces

**Reply by LevYas** [reply]
Would be glad to see it in the Analogy! For inspiration, here's a timeline view in AppInsights:
![image](https://github.com/user-attachments/assets/8fd87457-ab08-49cf-9e19-8fb89dc7754d)

There's also an event view, but it just lists all the entries that happened in this operation and its children.
Bot these views appear after I click on a log entry. If it's part of an operation, it correlates the entry with other entries in this operation

**Reply by LevYas** [reply]
I tried to find correlated entries by using "search everywhere" checkbox. But if I activate the filter, no records are shown at all. Am I using it wrong?
![image](https://github.com/user-attachments/assets/f887865e-b306-40d3-b6cf-203052231961)

**Reply by LiorBanai** [reply]
There is a bug regarding the search everywhere where it doesn't search everywhere 🙂

I'll fix that soon

**Reply by LevYas** [reply]
Ah, got it, thank you! Looking forward to it :)

**Reply by LiorBanai** [reply]
Hi @LevYas can you attach example file log with few lines that should be correlated between them. You can delete data in any other fields that are  sensitive. I would like to make sure I have valid data to work on it.

you can attach it to https://github.com/Analogy-LogViewer/Analogy.LogViewer/issues/2103 as I'm also working on the search everywhere right now.

and thanks for the Ko-fi donation. Much appreciated 😄

**Reply by LevYas** [reply]
Sure! I'd need some time to redact logs, I'll do it tomorrow and will attach to the linked issue.
You are welcome! Glad to make your day better.

**Reply by LiorBanai** [reply]
will be handled by: "Create hierarchical view between log entries" #2191
